### PR TITLE
Master

### PR DESCRIPTION
**Pull request by pointcontrols**
Package all dependencies on windows

### CODE REVIEW

File: WINDOWS_INSTALL.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+WINDOWS INSTALLATION INSTRUCTIONS
```
Comment (willpatera) [inline WINDOWS_INSTALL.txt]: Please change this file to `.md` file and format content accordingly.

File: WINDOWS_INSTALL.txt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+------------------------------------
+
+PREREQUISITES
+==============
+1. Libusb with isochronous support
```
Comment (willpatera) [inline WINDOWS_INSTALL.txt]: Please link to our fork of libusb with isochronous support

File: WINDOWS_INSTALL.txt [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+2. Libuvc
```
Comment (willpatera) [inline WINDOWS_INSTALL.txt]: Link to the repo please.

File: uvc.pyx [hunk 4]
```diff
@@ -573,7 +573,7 @@ cdef class Capture:
 
     def get_frame(self):
         cdef int status, j_width,j_height,jpegSubsamp,header_ok
-        cdef int  timeout_usec = 1000000 #1sec
```
Comment (willpatera) [inline uvc.pyx]: @mkassner - please review this change

Comment (mkassner) [inline uvc.pyx]: @pointcontrols  what is the reasoning behind this change?

**Pull request merged**